src/components/Overview.tsx

```diff
@@ -0,0 +1,35 @@
+import { View, StyleSheet } from 'react-native'
+
+import { StyledText } from './StyledText'
+import { RefreshIcon } from './icons/Icons'
+import { colors } from '../config/themes/appThemes'
+
+interface OverviewProps {
+  updateTime: string
+}
+
+export const Overview: React.FC<OverviewProps> = (props) => {
+  const { updateTime } = props
+
+  return (
+    <View>
```

Comment:
Usar este View para estilar; esto facilita poner un espacio entre los elementos
```suggestion
    <View style={styles.container}>
```